patches/@lightningjs__renderer@3.0.0-beta8.patch

```diff
@@ -0,0 +1,935 @@
+diff --git a/dist/src/core/CoreNode.js b/dist/src/core/CoreNode.js
```

Comment:
This patch is only necessary until this commit lands in v3 beta: https://github.com/lightning-js/renderer/commit/3cffb586c0b5dc1e958aa7764649746609523211